clio-server/src/main/scala/org/broadinstitute/clio/server/ClioServerConfig.scala

```diff
@@ -1,48 +1,63 @@
 package org.broadinstitute.clio.server
 
-import org.broadinstitute.clio.util.model.Env
+import org.broadinstitute.clio.util.json.ModelAutoDerivation
+import org.broadinstitute.clio.util.model.ServiceAccount
 
 import com.typesafe.config.Config
 import net.ceedubs.ficus.Ficus._
 import net.ceedubs.ficus.readers.ArbitraryTypeReader._
 import net.ceedubs.ficus.readers.ValueReader
 
+import scala.collection.JavaConverters._
 import scala.concurrent.duration.FiniteDuration
 
+import java.io.File
+import java.nio.file.{Files, Path}
+
 /**
   * Configuration file accessors.
   */
 object ClioServerConfig {
-  private val config = ClioConfig.getConfig("server")
+  private val server = ClioConfig.getConfig("server")
+
+  object Persistence extends ModelAutoDerivation {
+    implicit val pathReader: ValueReader[Path] =
+      (config: Config, path: String) => new File(config.getString(path)).toPath
 
-  object Environment {
-    implicit val envReader: ValueReader[Env] = new ValueReader[Env] {
-      override def read(config: Config, path: String): Env = {
-        val maybeEnv = config.as[String](path)
-        Env.withName(maybeEnv)
+    private val persistence = server.as[Config]("persistence")
+    private val jsonPath = persistence.getAs[Path]("service-account-json")
+    lazy val serviceAccount: Option[ServiceAccount] =
+      jsonPath.map { path =>
+        import io.circe.parser._
+        val jsonBlob =
+          Files.readAllLines(path).asScala.mkString("\n").stripMargin
```

Comment:
Why not Source.fromFile(path.toFile).mkString ?

Reply:
Not sure how it's done in scala, but in Java I'd do `new String(Files.readAllBytes(p))`